src/main/java/ch/wisv/areafiftylan/util/TestDataRunner.java

```diff
@@ -66,6 +70,12 @@ public void run(String... evt) throws Exception {
         ticketRepository.save(ticket2);
         ticketRepository.save(ticket3);
 
+        Team team = new Team("testTeam", testUser1);
+        team.addMember(testUser2);
+        team.addMember(testUser3);
+
+        teamRepository.save(team);
```

Comment:
It's just the TestDataRunner, it's not essential that's flushed right there. It's been working fine :)